### DISCUSSION TITLE
What is the difference between EasyAdmin and ApiPlatform Admin ?

### DISCUSSION THREAD

**Discussion by BliiTzZ** [symfony/symfony#48361] [Q&A] [answered]
Hello,

I am setting up a toy library project, and during my research, I came across easyadmin, which seemed to be a good solution to create my administration interface for employees. 

But I also need to use ApiPlatform to create an api linked to a react application for the users. 

So I would have the internal part with the dashboard for the admins

and the web client part with react making calls to the api

But in my research I found that there is ApiPlatform Admin which allows to create easily an admin dashboard with React, and I try to understand what is the difference with easyadmin, in which case to use which solution ? What is the best way to integrate my dashboard for employees?

**Comment by nalabdou** [comment, marked as answer]
Howdy @BliiTzZ 

EasyAdmin and ApiPlatformAdmin may be shared the Admin suffix, but they are two different bundles.

[EasyAdmin](https://github.com/EasyCorp/EasyAdminBundle) creates backend administration panels it could be used to create administration [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete)
using procedural way with forms and full forms submissions, no API calls, only good old HTTP calls with POST and GET requests.

[ApiPlatformAdmin](https://api-platform.com/docs/admin/) creates frontend administration panels,
so all the [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) operations uses API calls with POST, PUT, PATCH, DELETE and GET requests.

EasyAdmin is perfect if you don't want an API and you want to manage your backend,
it's rendered with PHP and Symfony routing.
ApiPlatformAdmin is good if you want an API with full controls from frontend dashboard entirely rendered client-side in JS/React.

Best regards.
